### PR TITLE
fix json and display issue 

### PR DESCRIPTION
**Pull request by leeannachen**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -29,8 +29,69 @@ function addRandomGreeting() {
...
+  taskElement.appendChild(deleteButtonElement);
+  return taskElement;
+}
+
+// Tells the server to delete the task. 
+function deleteTask(task) {
+  const params = new URLSearchParams();
+  params.append('id', task.id);
+  fetch('/delete-task', {method: 'POST', body: params});
+}
```
Comment (aokeke) [inline portfolio/src/main/webapp/script.js]: I don't see the servlet for these in this PR. I'd either add it to this PR or remove this from the PR,

**Review by aokeke — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -29,8 +29,76 @@ function addRandomGreeting() {
...
 }
 
-async function getRandomQuoteUsingAsyncAwait() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('quote-container').innerText = quote;
+// Fetches comments
+function getComments() {
+    console.log("Fetching comments")
+    const responsePromise = fetch('/data');
```
Comment (aokeke) [inline portfolio/src/main/webapp/script.js]: should this be `fetch('/list-comments')`?

**Review by aokeke — CHANGES_REQUESTED** [review]
Overall this looks really good. One small thing I want you to double check.

This structure is fine, but if I were designing something like this, I would have used one servlet with a url `/comments` and just implemented the `doPost`, `doGet`, and `doDelete` methods all in that one servlet. The deleting comments would use the DELETE verb rather than the POST verb. Then for editing a comment you can implement `doPut`. But in the training I guess they wanted you to practice having multiple servlets.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (leeannachen) [reply portfolio/src/main/webapp/script.js]: That was some leftover code from one of the walk-through;  I will delete that.

**Pull request merged**